### ISSUE TITLE
missing: rendertype_glint_direct

### ISSUE THREAD

**Issue by McTsts** [McTsts/Minecraft-Shaders-Wiki#4]

**Comment by McTsts** [comment]
Tested with:
- Item: Diamond Sword, Stick, Nether Star (1st PoV held, 3rd PoV held, dropped, item frame, inventory)
- Block: Stone, Oak Leaves (1st PoVheld, 3rd PoV held, dropped, item frame, inventory)
- Glass: Glass, Lime Stained Glass, Lime Stained Glass Pane (1st PoV held, 3rd PoV held, dropped, item frame, inventory)
- Armor: Leather Chestplate, Diamond Chestplate (1st PoVheld, 3rd PoV held, dropped, item frame, armor stand, worn, inventory)

Affects:
- Items (held, dropped, item frame, inventory)
- Blocks (held, dropped, item frame, inventory)
- Glass (1st pov held, inventory)
- Armor (held, dropped, item frame, inventory)

Basically everything except glass in world (item frame, dropped, 3rd pov held) and worn armor
